js/app/Data/BaseTiledFormat.js

```diff
@@ -267,7 +267,8 @@ define([
     search: function(query, offset, limit, cb) {
       var self = this;
 
-      var url = self.getUrl("search", data, -1) + "/search?query=" + encodeURIComponent(query);
+      var key = {query: query, offset: offset, limit: limit};
+      var url = self.getUrl("search", key, -1) + "/search?query=" + encodeURIComponent(query);
```

Comment:
No, the data hash is needed to generate a key for url round robin. That is what we broke as "data" is not defined in this context